GTG/gtk/editor/editor.py

```diff
@@ -258,24 +258,24 @@ def refresh_editor(self, title=None, refreshtext=False):
         if status == Task.STA_DISMISSED:
             self.donebutton.set_label(GnomeConfig.MARK_DONE)
             self.donebutton.set_tooltip_text(GnomeConfig.MARK_DONE_TOOLTIP)
-            self.donebutton.set_icon_name("gtg-task-done")
+            self.donebutton.set_icon_name(Gtk.STOCK_APPLY)
             self.dismissbutton.set_label(GnomeConfig.MARK_UNDISMISS)
             self.dismissbutton.set_tooltip_text(undismiss_tooltip)
-            self.dismissbutton.set_icon_name("gtg-task-undismiss")
+            self.dismissbutton.set_stock_id(Gtk.STOCK_REFRESH)
```

Comment:
Again, why to combine set_icon_name and set_stock_id for the same purpose? Pick single way how to set icons. Consistency makes the code much readable